starter-code/public/scripts/controllers/routes.js

```diff
@@ -2,5 +2,7 @@
 
 // TODO: Configure routes for this app with page.js, by registering each URL your app can handle,
 // linked to a a single controller function to handle it:
-
+page('/', window.articleController.index);
```

Comment:
Nice work. You dont actually need to attach the window to this function. Just having articleController.index will suffice.